models/articles.js

```diff
@@ -2,6 +2,18 @@
 const db = require("../db/connection");
 
 // Model Funcs
+exports.selectArticles = async () => {
+  const result = await db.query(
```

Comment:
This is spot-on... have you considered deconstructing rows on this line? It only saves you a few characters on line 14, but it could be neat! What do you think?

Reply:
ahhh thanks Joey, so it would be something like 'const { rows } =...' on line 5, then just return 'rows' on line 14?